common/api/appui-react.api.md

```diff
@@ -795,7 +795,15 @@ export enum ColorTheme {
     // (undocumented)
     Dark = "dark",
     // (undocumented)
-    Light = "light"
+    HighContrastDark = "high-contrast-dark",
```

Comment:
This might not be great if a custom high contrast theme is needed (specifically an iTwinUI `themeOption`).

Reply:
iTwinUI is not offering that kind of random naming option, and we no longer have any colors of our own, if someone want to create a custom theme for iTwinUI, they need to override their color vars, and I doubt that people will go through and create "light" "dark" and high contrast variant of their theme. (If they do, they should probably use `"Inherit"` anyway)

At this point, our `ThemeManager` is no longer necessary but to preserve compatibility with someone who might use overrides in our theme, I've left it as is (updating the `<html data-theme={...}>` attribute  . Adding "high contrast" as a separate feature/setting will add to the complexity whereas I'd rather completely remove this theme system altogether.